packages/terra-badge/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 3.61.0 - (February 20, 2024)
```

Comment:
I see this package and several others don't seem to have any changes other than "Minor dependency version bump" and no updates to package.json... were there just updates that didn't have changelog entries?

Reply:
Ok, I see for `terra-badge` at least there were some updates: 
<img width="1301" alt="image" src="https://github.com/cerner/terra-core/assets/42594047/d52678b8-0bfb-40fb-b31f-3b5d7b709eeb">

I think these changes also apply to the other packages in this release so disregard my above comment.

Reply:
Yeah, this release is mainly adding the missing peerDependency for several packages.